network-api/networkapi/mozfest/templates/fragments/hero/featured_video.html

```diff
@@ -38,7 +38,6 @@
                 {% if block.block_type == 'external_video' %}
                     <iframe
                         src=""
-                        frameborder="0"
```

Comment:
<img width="1271" alt="Screenshot 2024-02-13 at 12 21 51" src="https://github.com/MozillaFoundation/foundation.mozilla.org/assets/35934884/059d70b6-479d-4a83-9b69-1a4a1da3ab24">

I have removed this as it was flagged as an error instead of a warning. 

This is a deprecated attribute, marked as "obsolete" since browsers no longer support it and styling for the iframe border is applied via the CSS border property now.

The site's CSS does apply `border: 0` to iframes.


<img width="274" alt="Screenshot 2024-02-13 at 12 24 50" src="https://github.com/MozillaFoundation/foundation.mozilla.org/assets/35934884/d26eccbf-42e0-4d2b-8a95-c4dc9356c17a">
<img width="274" alt="Screenshot 2024-02-13 at 12 24 44" src="https://github.com/MozillaFoundation/foundation.mozilla.org/assets/35934884/2cace54e-1230-4976-a5c5-5d1c8d7cddad">

I imagine there might be the case where older browsers like IE could end up drawing a border around the iframe, however it should be pretty safe to remove as the only negative would be legacy browsers applying their default browser styling.

Reply:
Agree!